Description: direct link to documentation
 It's not in CVS anymore.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: https://bugs.debian.org/710377
Last-Update: 2019-10-06

---

--- graphviz-2.42.2.orig/cmd/dot/osage.1
+++ graphviz-2.42.2/cmd/dot/osage.1
@@ -54,7 +54,7 @@ Traditionally, osage supports the follow
 that has a non\(hynull "href" attribute.),
 \fB\-Tcmapx\fP (client\(hyside imagemap for use in html and xhtml).
 Additional less common or more special\(hypurpose output formats
-can be found at http://www.graphviz.org/cvs/doc/info/output.html.)
+can be found at http://www.graphviz.org/doc/info/output.html.)
 .P
 Alternative plugins providing support for a given output format
 can be found from the error message resulting from appending a ':' to the format. e.g. \fB-Tpng:\fP
--- graphviz-2.42.2.orig/cmd/dot/patchwork.1
+++ graphviz-2.42.2/cmd/dot/patchwork.1
@@ -48,7 +48,7 @@ Traditionally, patchwork supports the fo
 that has a non\(hynull "href" attribute.),
 \fB\-Tcmapx\fP (client\(hyside imagemap for use in html and xhtml).
 Additional less common or more special\(hypurpose output formats
-can be found at http://www.graphviz.org/cvs/doc/info/output.html.)
+can be found at http://www.graphviz.org/doc/info/output.html.)
 .P
 Alternative plugins providing support for a given output format
 can be found from the error message resulting from appending a ':' to the format. e.g. \fB-Tpng:\fP
